cpp/examples/lct_secir.cpp

```diff
@@ -126,6 +69,109 @@ int main()
     model.parameters.get<mio::lsecir::CriticalPerSevere>()              = 0.25;
     model.parameters.set<mio::lsecir::DeathsPerCritical>(0.3);
 
+    if (use_initializer_flows) {
+        // Example how to use the class Initializer for the definition of an initial vector for the LCT model.
+        // Create TimeSeries with num_transitions elements.
+        int num_transitions = (int)mio::lsecir::InfectionTransition::Count;
+        mio::TimeSeries<ScalarType> flows(num_transitions);
+
+        ScalarType dt = 0.1;
+
+        mio::TimeSeries<ScalarType>::Vector vec_flows(num_transitions);
+        vec_flows[(int)mio::lsecir::InfectionTransition::SusceptibleToExposed]                 = 2.0;
+        vec_flows[(int)mio::lsecir::InfectionTransition::ExposedToInfectedNoSymptoms]          = 1.0;
+        vec_flows[(int)mio::lsecir::InfectionTransition::InfectedNoSymptomsToInfectedSymptoms] = 8.0;
+        vec_flows[(int)mio::lsecir::InfectionTransition::InfectedNoSymptomsToRecovered]        = 4.0;
+        vec_flows[(int)mio::lsecir::InfectionTransition::InfectedSymptomsToInfectedSevere]     = 1.0;
+        vec_flows[(int)mio::lsecir::InfectionTransition::InfectedSymptomsToRecovered]          = 4.0;
+        vec_flows[(int)mio::lsecir::InfectionTransition::InfectedSevereToInfectedCritical]     = 1.0;
+        vec_flows[(int)mio::lsecir::InfectionTransition::InfectedSevereToRecovered]            = 1.0;
+        vec_flows[(int)mio::lsecir::InfectionTransition::InfectedCriticalToDead]               = 1.0;
+        vec_flows[(int)mio::lsecir::InfectionTransition::InfectedCriticalToRecovered]          = 1.0;
+        // Add initial time point to time series.
+        flows.add_time_point(-110, vec_flows);
+        // Add further time points until time 0.
+        while (flows.get_last_time() < -1e-10) {
+            flows.add_time_point(flows.get_last_time() + dt, vec_flows);
```

Comment:
I think here you should make vec_flows dependent on dt as in the IDE example. For example when I set `dt=0.001` this leads to an error.

Reply:
Yes, thanks a lot!